sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSqlTypesUtils.java

```diff
@@ -24,6 +24,11 @@
 @Internal
 public class ZetaSqlTypesUtils {
 
+  public static final BigDecimal NUMERIC_MAX_VALUE =
```

Comment:
nit: these constants are only used in `ZetaSqlMathFunctionsTest.java`, do they belong there?